tools/github/autoresponder.ts

```diff
@@ -120,7 +122,22 @@ console.log("similar discussions:", similarDiscussions);
 const answerFromAi = await getAnswerFromAi(API_SECRET, DISCUSSION_BODY);
 console.log("response from AI:", answerFromAi.trim().slice(0, 300));
 
-const fullBody = `${commentHeader}${answerFromAi}\n\n---\n\nHere are some similar discussions:\n\n${similarDiscussions}`;
+const hasAnswerFromAi = answerFromAi !== "Sorry, I don't know how to help with that.";
+const hasSimilarDiscussions = similarDiscussions.length > 0 && similarDiscussions !== "No similar discussions found.";
+
+if (!hasSimilarDiscussions && !hasAnswerFromAi) {
+  // @ts-expect-error deno
+  Deno.exit(0);
+}
+
+let fullBody = `${commentHeader}`;
+
+if (hasAnswerFromAi) {
+  fullBody = `${fullBody}${commentSeparator}${answerFromAi}`;
+}
+if (hasSimilarDiscussions) {
+  fullBody = `${fullBody}${commentSeparator}Here are some similar discussions:\n${similarDiscussions}`;
```

Comment:
Suggestion: Tempted to make this sound somewhat more conversational. Just on a new line without the separator, it might say something like 

> "I did a search, and I managed to find these other Discussions that might be similar or related to your question. Be sure to check them out."